egs/wsj/s5/steps/cleanup/segment_long_utterances.sh

```diff
@@ -0,0 +1,389 @@
+#!/bin/bash
+
+# Copyright 2014  Guoguo Chen
+#           2016  Vimal Manohar
+# Apache 2.0
+
+set -e 
+set -o pipefail
+
+. path.sh
+
+set -u
+
+# Uniform segmentation options
+max_segment_duration=30
+overlap_duration=5
+
+# Decode options
+graph_opts=
+nj=4
+lmwt=10
+
+# TF-IDF similarity search options
+max_words=1000
+num_neighbors_to_search=1
+neighbor_tfidf_threshold=0.5
+
+# First-pass segmentation opts
+min_segment_length=0.5
+min_new_segment_length=1.0
+max_tainted_length=0.05
+max_edge_silence_length=0.5
+max_edge_non_scored_length=0.5
+max_internal_silence_length=2.0
+max_internal_non_scored_length=2.0
+unk_padding=0.05
+max_junk_proportion=0.1
+min_split_point_duration=0.1
+max_deleted_words_kept_when_merging=1
+silence_factor=1
```

Comment:
If some of these configuration options turn out to be not that useful, and you plan to always leave them at a certain value, it might be a good idea to remove them from the scripts and maybe the code.  I like to prune away non-useful options to keep the scripts uncluttered.